website/docs/docs/build/snapshots.md

```diff
@@ -587,6 +587,16 @@ Snapshot results:
 
 </details>
 
+<VersionBlock firstVersion="1.9">
+
+### Define column names in snapshot table
+
+In dbt Cloud, there's a config which will now allow you to define the names of the columns that are created in the [snapshot table](/docs/build/snapshots#snapshot-meta-fields). Users can now set this config to use different names.
```

Comment:
i'm a little confused -- is this only for dbt cloud? i thought this was something avail to both cloud and core users since it relates to snapshots? maybe the diff is it's available to versionless users now and 1.9 users when 1.9 is launched?

Reply:
Sorry - that was a force if habit. I'll update this. Thanks for this